sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -516,8 +516,8 @@ public void flush(long timeoutMillis) {
   @ApiStatus.Internal
   @Override
   public @NotNull SentryId captureTransaction(
-      final @NotNull ITransaction transaction, final @Nullable Object hint) {
-    Objects.requireNonNull(transaction, "transaction is required");
+      final @NotNull SentryTracer tracer, final @Nullable Object hint) {
```

Comment:
I was thinking we'd keep `captureTransaction(Transaction ..)`
`SentryTracer.finish` would in fact do:

```
var tran = new SentryTransaction() { ..tags = self.tags }
hub.captureTransaction(tran);
```

Reply:
yeah me too

Reply:
is there a reason to take `SentryTracer` instead of the `Transaction` itself? the reason is, the reason I ask is, we'd not need ot change SentryClient

Reply:
We need to clear the transaction from the scope on capture, since on the scope we will have instance of a Span (likely SentryTracer), not a transaction we need to pass the original captured transaction.